lib/commands/debug/python-debug.js

```diff
@@ -0,0 +1,31 @@
+const { execSync } = require('child_process');
+const AbstractDebugFlow = require('./abstract-debug-flow');
+const helper = require('./helper');
+
+class PythonDebugFlow extends AbstractDebugFlow {
+    static canHandle(runtime) {
+        return runtime.includes('python');
+    }
+
+    constructor({ skillInvocationInfo, waitForAttach, debugPort, token, skillId, debugRegion }) {
+        const localDebuggerPath = helper.getPythonDebugAdapterPath();
```

Comment:
should getPythonDebugAdapterPath part of this class?

Reply:
Moved to PythonRunFlow class's constructor